pkg/config/config_parser.go

```diff
@@ -127,6 +127,8 @@ func decodeCloudQueryBlock(block *hcl.Block, ctx *hcl.EvalContext) (CloudQuery,
 	cq.Logger = &logging.GlobalConfig
 	var diags hcl.Diagnostics
 	diags = diags.Extend(gohcl.DecodeBody(block.Body, ctx, &cq))
+	// We don't care about additional configuration blocks, so just ignore them
+	cq.Remain = nil
```

Comment:
A possible improvement can be to print warnings on unexpected configuration blocks, migrating blocks that changed, etc.